### DISCUSSION TITLE
Question about `interactive-examples.mdn.mozilla.net`

### DISCUSSION THREAD

**Discussion by erosman** [mdn/mdn-community#765] [Content]
At the moment, I am using CM5 in a WebExtension but it is becoming outdated and I have been trying to update it. I haven't found a good guide on compiling the CM 6 for JS & CSS (only) plus corresponding linters (I have been asking around for a while).

I then noticed the `interactive-examples.mdn.mozilla.net` on MDN and  would like to enquire about its  `codemirror.js` & `editor.js` compiling process.

**Comment by pransh15** [comment]
Hi @erosman, nice to see you here! 👋 
I'll share the discussion with the team so you can get answers. 👍

**Comment by bsmth** [comment]
Thanks for asking. The PR to upgrade CM5 -> CM6 was done here: https://github.com/mdn/bob/pull/907

Maybe that's useful for you. Do you have any links to share or other specific questions / problems?

**Comment by erosman** [comment]
Thank you @bsmth 

I use CM5 in [FireMonkey](https://github.com/erosman/[firemonkey](https://github.com/erosman/firemonkey)) extension.
I have no experience in compiling and bundling which is needed for CM6.

I would need a step-by-step guide in how to generate the files from CodeMirro6 with JS & CSS modes along with ESLint & Stylelint.

_PS. Add-ons review team would also need the same instructions in order to be able to verify the generated code._

> [Submission Guidelines](https://extensionworkshop.com/documentation/publish/add-on-policies/#submission-guidelines)
> 
> Code must be provided in a way that is reviewable. Add-ons may contain transpiled, minified or otherwise machine-generated code, but Mozilla needs to review a copy of the source code before any of these steps have been applied. The author must provide this information to Mozilla during submission along with instructions on how to reproduce the build. Build tools or environments that no longer appear to be supported by their maintainers are not accepted. Reviewers may ask you to refactor parts of the code if it is not reviewable.
